recipe/meta.yaml

```
@@ -10,11 +10,12 @@ source:
   sha256: 24949046643865445c591c79114f2ee3cd48eb2650f6ff9f20b4890b5ee9a94b
 
 build:
-  number: 2
+  number: 3
   skip: true  # [win and vc<14]
 
 requirements:
   build:
+    - make
```

Sorry, I should have mentioned that in my initial comment, but you might want to guard this as
```suggestion
    - make  # [unix]
```

---

Thanks!